test/Masa.Utils.Caller.Core.Tests/CallerTest.cs

```diff
@@ -44,4 +42,135 @@ public void TestCaller()
         Assert.IsTrue(caller.GetType().FullName != daprCaller.GetType().FullName);
         Assert.IsTrue(caller.GetType().FullName == httpCaller.GetType().FullName);
     }
+
+    [TestMethod]
+    public void TestMultiDefaultCaller()
+    {
+        IServiceCollection services = new ServiceCollection();
+        Assert.ThrowsException<ArgumentException>(() =>
+        {
+            services.AddCaller(opt =>
+            {
+                opt.UseHttpClient(builder =>
+                {
+                    builder.Name = "github";
+                    builder.BaseApi = "https://github.com/masastack";
+                    builder.IsDefault = true;
+                });
+                opt.UseHttpClient(builder =>
+                {
+                    builder.Name = "gitee";
+                    builder.BaseApi = "https://gitee.com/masastack";
+                    builder.IsDefault = true;
+                });
+            });
+        });
+    }
+
+    [TestMethod]
+    public void TestMultiDefaultCaller2()
+    {
+        IServiceCollection services = new ServiceCollection();
+        Assert.ThrowsException<ArgumentException>(() =>
+        {
+            services.AddCaller(opt =>
+            {
+                opt.UseHttpClient(builder =>
+                {
+                    builder.Name = "gitee";
+                    builder.BaseApi = "https://gitee.com/masastack";
+                    builder.IsDefault = true;
+                });
+            });
+            services.AddCaller(opt =>
+            {
+                opt.UseHttpClient(builder =>
+                {
+                    builder.Name = "github";
+                    builder.BaseApi = "https://github.com/masastack";
+                    builder.IsDefault = true;
+                });
+            });
+        });
+    }
+
+    [TestMethod]
+    public void TestRepeatCallerName()
+    {
+        IServiceCollection services = new ServiceCollection();
+        Assert.ThrowsException<ArgumentException>(() =>
+        {
+            services.AddCaller(opt =>
+            {
+                opt.UseHttpClient(builder =>
+                {
+                    builder.Name = "gitee";
```

Comment:
gitee?

Reply:
Here is the name of the caller, which has no practical effect. For the convenience of understanding, it was renamed giteeCaller